### PR TITLE
[ADD][purchase_partial_invoicing] Add the possibility to define some quantity to not invoice

### PR DESCRIPTION
**Pull request by adrienpeiffer**
- Add a new wizard in More menu to set some quantity as not invoiceable
- Replace fully invoice domain with a default filter
- Add a filter to see only line with cancelled quantity
- Add tests

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1: Thanks for the improvement, although it is not synchronized from picking information. Maybe you can add this in known issues / roadmap section

File: purchase_partial_invoicing/tests/test_purchase_partial_invoicing.py [hunk 1]
```diff
@@ -206,3 +207,73 @@ def test_invoiced_status(self):
         self.assertTrue(line_to_partial.invoiced)
         # I check if the purchase order is flag as invoiced
         self.assertTrue(self.purchase_order.invoiced)
+
+    def test_cancel_quantity(self):
+        self.common_test()
+        quantity_to_invoiced = 4
```
Comment (eLBati) [inline purchase_partial_invoicing/tests/test_purchase_partial_invoicing.py]: `quantity_to_invoice` better?

**Comment by eLBati** [comment]
:+1: thanks

**Comment by adrienpeiffer** [comment]
@eLBati @pedrobaeza Thanks for the review !

**Comment by sbidoul** [comment]
:+1:

**Pull request merged**